### PR TITLE
allow to run migrate script from source AGPUSH-1527

### PR DESCRIPTION
**Pull request by lfryc**
https://issues.jboss.org/browse/AGPUSH-1527

### CODE REVIEW

**Comment by lfryc** [comment]
@matzew @danbev could you please review?

File: migrator/README.md [hunk 1]
```diff
@@ -52,7 +52,16 @@ In case of a successful run, the script prints
...
 
     ./bin/ups-migrator --logLevel=DEBUG update
+    
+#### Running the script from source
 
+If you are developing UnifiedPush Server, you can run the script from source instead of from distribution:
+
+    cd aerogear-unifiedpush-server/migrator/
+    mvn clean install
+    mvn dependency:copy-dependencies -DincludeScope=runtime
```
Comment (lfryc) [inline migrator/README.md]: should we make it `mvn clean install dependency:copy-dependencies -DincludeScope=runtime` instead?

**Comment by matzew** [comment]
the change from `644` to `755` on:
* migrator/src/main/shell/ups-migrator 

not sure, the source code should be executable, tbh

**Comment by lfryc** [comment]
@matzew right, I had `sh ./src/main/shell/ups-migrator update` first, but it is platform dependent.. if we forget about windows, then we can do that...

**Comment by matzew** [comment]
yep - windows - we need a BAT. Sebi has a ticket for that, for future or
something like that

On Thu, Nov 19, 2015 at 12:17 PM, Lukáš Fryč <notifications@github.com>
wrote:

> @matzew <https://github.com/matzew> right, I had sh
> ./src/main/shell/ups-migrator update first, but it is platform
> dependent.. if we forget about windows, then we can do that...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/642#issuecomment-158026380>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

File: migrator/README.md (hunk 1 above)
Comment (danbev) [inline migrator/README.md]: Would it be worth having a a profile that does this, like ```mvn -PsetupEnv``` or something like that?

**Comment by danbev** [comment]
:+1: Works like advertised.

**Comment by lfryc** [comment]
Thanks for testing @danbev ,

I've incorporated README change suggested by @matzew and merged mvn to one-liner..

**Comment by matzew** [comment]
@lfryc I still see the 644 -> 755 change on the migrator source file - should we really do that ?

**Comment by lfryc** [comment]
Yep, sry, nice catch, lemme fix that

**Comment by lfryc** [comment]
@matzew should be fine now

**Comment by matzew** [comment]
:+1:

**Pull request merged**